src/legacy/ui/ui_bundles/app_entry_template.js

```diff
@@ -38,6 +38,10 @@ import 'childnode-remove-polyfill';
 import { i18n } from '@kbn/i18n';
 import { CoreSystem } from '__kibanaCore__'
 
+// Load all Kibana styles
+const context = require.context('ui/styles', false, /[\\/\\\\](?!mixins|variables|_|\\.|bootstrap_(light|dark))[^\\/\\\\]+\\.less/);
+context.keys().forEach(key => context(key));
```

Comment:
I was not able to move this into Core's source code directly due to a Webpack limitation with `require.context`.

Reply:
I just had this problem https://github.com/elastic/kibana/issues/48883
I run your branch locally and it failed with 
`Can't resolve 'ui/autoload/styles' in '/Users/mikhailshustov/work/kibana/plugins/my_plugin_name/public'`
It means that all 3rd party plugins will be broken in the next release 😞 probably we can make this file empty for LP plugins?